examples/AspNetCore/Controllers/WeatherForecastController.cs

```diff
@@ -14,58 +14,42 @@
 // limitations under the License.
 // </copyright>
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Net.Http;
-using Examples.AspNetCore.Models;
+namespace Examples.AspNetCore.Controllers;
+using System.Diagnostics;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Extensions.Logging;
 
-namespace Examples.AspNetCore.Controllers
+[ApiController]
+[Route("[controller]")]
+public class WeatherForecastController : ControllerBase
 {
-    [ApiController]
-    [Route("[controller]")]
-    public class WeatherForecastController : ControllerBase
+    private static readonly string[] Summaries = new[]
     {
-        private static readonly string[] Summaries = new[]
-        {
-            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching",
-        };
-
-        private static readonly HttpClient HttpClient = new HttpClient();
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching",
+    };
 
-        private readonly ILogger<WeatherForecastController> logger;
+    private readonly ILogger<WeatherForecastController> logger;
+    private readonly ExampleMeter exampleMeter;
 
-        public WeatherForecastController(ILogger<WeatherForecastController> logger)
-        {
-            this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
-        }
+    public WeatherForecastController(ILogger<WeatherForecastController> logger, ExampleMeter exampleMeter)
+    {
+        this.logger = logger;
+        this.exampleMeter = exampleMeter;
+    }
 
-        [HttpGet]
-        public IEnumerable<WeatherForecast> Get()
+    [HttpGet(Name = "GetWeatherForecast")]
+    public IEnumerable<WeatherForecast> Get()
+    {
+        using var activity = Tracing.ActivitySource.StartActivity("WeatherForecast GET", ActivityKind.Internal);
```

Comment:
Also - this doesn't really show any additional value... activity/span is already created, so why would user create a new one? Some comments to indicate that this is intended to demonstrate how once create more activity for any sub-operation.

Another useful example would be to show Activity.Current.SetTag to add additional tags to the activity.